<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qRIMACdlg</class>
 <widget class="QDialog" name="qRIMACdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qRIMACdlg</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>511</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Transfertattributs">
    <attribute name="title">
     <string>Transfert attributs</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>326</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="pir">
          <property name="text">
           <string>NUAGE PIR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PIR_IN"/>
        </item>
        <item>
         <widget class="QToolButton" name="PIR_IN_SEARCH">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="swir">
          <property name="text">
           <string>NUAGE SWIR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SWIR_IN"/>
        </item>
        <item>
         <widget class="QToolButton" name="SWIR_IN_SEARCH">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="rvb">
          <property name="text">
           <string>NUAGE RVB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RVB_IN"/>
        </item>
        <item>
         <widget class="QToolButton" name="RVB_IN_SEARCH">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="select_cloud">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sélection des nuages:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lancer">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>290</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Lancer le transfert d'attributs</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Classification">
    <attribute name="title">
     <string>Classification</string>
    </attribute>
    <widget class="QLabel" name="type_classif">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type de classification:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="button_non_supervisee">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Non supervisée</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="button_supervisee">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Supervisée </string>
     </property>
    </widget>
    <widget class="QPushButton" name="lancer_classif">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>241</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Lancer la classification</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>391</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="nuage_classif">
        <property name="text">
         <string>Nuage à classifier :  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="choix"/>
      </item>
      <item>
       <widget class="QToolButton" name="choix_nuage">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>188</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="algo">
        <property name="text">
         <string>Algorithme :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="choix_algo">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>...</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>k-means</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>182</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="nb_classes">
        <property name="text">
         <string>Nombre de classes :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="choix_nb_classes">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>175</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="nb_iterations">
        <property name="text">
         <string>Nombre itérations :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="choix_nb_iterations">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="classif">
        <property name="text">
         <string>Classification :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="out_classif"/>
      </item>
      <item>
       <widget class="QToolButton" name="out_explorateur">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Vérité terrain</string>
    </attribute>
    <widget class="QPushButton" name="select_point">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Sélectionner des points </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>281</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter la sélection à la vérité terrain </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler la sélection </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>281</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Enregistrer la vérité terrain</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>381</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sélection du nuage :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="VT"/>
      </item>
      <item>
       <widget class="QToolButton" name="VT_nuage">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>381</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nom de la classe :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>381</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nom de sortie :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_2">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
